R/simulateConsumerResource.R

```diff
@@ -46,34 +46,20 @@
 #' @examples
 #' n_species <- 2
 #' n_resources <- 4
-#' ExampleCR <- simulateConsumerResource(
+#' tse <- simulateConsumerResource(
 #'     n_species = n_species,
 #'     n_resources = n_resources
 #' )
 #'
-#' miaViz::plotSeries(ExampleCR, x = "time")
-#' # convert to relative abundance
-#' ExampleCR <- mia::transformCounts(ExampleCR, method = "relabundance")
-#' miaViz::plotSeries(ExampleCR, x = "time")
-#'
-#' # example to get relative abundance and relative proportion of resources
-#' n_species <- 2
-#' n_resources <- 4
-#' ExampleCR <- simulateConsumerResource(
-#'     n_species = n_species,
-#'     n_resources = n_resources, norm = TRUE
-#' )
-#'
 #' # example with user-defined values (names_species, names_resources, E, x0,
 #' # resources, growth_rates, error_variance, t_end, t_step)
-#' n_species <- 4
-#' n_resources <- 6
+#'
 #' ExampleE <- randomE(
 #'     n_species = n_species, n_resources = n_resources,
 #'     mean_consumption = 3, mean_production = 1, maintenance = 0.4
 #' )
 #' ExampleResources <- rep(100, n_resources)
-#' ExampleCR <- simulateConsumerResource(
+#' tse_1 <- simulateConsumerResource(
```

Comment:
could we have just `tse1`etc. instead of `tse_1` (by convention)? Here and elsewhere.